README.md

```
@@ -1,70 +1,96 @@
+  <img src="https://github-readme-stats.vercel.app/api?username=mgmgpyaesonewin&show_icons=true&theme=dracula" />
 </p>
 
+---
+
+### 🏆 GitHub Trophies
+
 <p align="center">
-  <a href="https://app.daily.dev/pyaesonewin">
-    <img src="https://api.daily.dev/devcards/ecdc02169f8f47109380b815df096438.png?r=0ej" width="400" alt="Pyae Sone Win's Dev Card"/>
+  <a href="https://github.com/ryo-ma/github-profile-trophy">
+    <img src="https://github-profile-trophy.vercel.app/?username=mgmgpyaesonewin&theme=chalk" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)